server/server_test.go

```diff
@@ -33,15 +36,53 @@ func TestListenAndServe(t *testing.T) {
 	}
 }
 
+func TestLogging(t *testing.T) {
```

Comment:
```suggestion
func TestMiddleware(t *testing.T) {
```

(We may want to verify other properties of the middleware later.)